tasks/sign-binaries/sign-binaries.yaml

```diff
@@ -42,36 +45,203 @@ spec:
     - name: windows-ssh-key-vol
       secret:
         secretName: $(params.windowsSSHKey)
+    - name: mac-ssh-key-vol
+      secret:
+        secretName: $(params.macSSHKey)
   workspaces:
     - name: data
       description: Workspace to save the results to
   results:
+    - name: unsignedMacDigest
+      type: string
+      description: |
+        Digest used by Mac signing host to pull unsignged content via ORAS
+    - name: signedMacDigest
+      type: string
+      description: |
+        Digest used to pull signed Mac content back to pipeline via ORAS
     - name: unsignedWindowsDigest
       type: string
       description: |
-        Digest used by signing host to pull unsignged content via ORAS
+        Digest used by Windows signing host to pull unsignged content via ORAS
     - name: signedWindowsDigest
       type: string
       description: |
-        Digest used to pull signed content back to pipeline via ORAS
+        Digest used to pull signed Windows content back to pipeline via ORAS
   steps:
     - name: push-unsigned-using-oras
       image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
       script: |
-        #!/usr/bin/env bash
+        #!/bin/bash
```

Comment:
Why this change? It doesn't really make a difference, but we use `#!/usr/bin/env bash` everywhere, so it's nice to stick with that.

Reply:
leftover code from before, updated

Reply:
ok. but the mocks should also use the same shebang as the actual task scripts.

Reply:
ack, shebang updated for tests.